### ISSUE TITLE
melanie2/Roundcube-Plugin-Mobile  no more works with new beta version 1.3.0 of Roundcube

### ISSUE THREAD

**Issue by Melkin1968** [messagerie-melanie2/Roundcube-Plugin-Mobile#25]
Hello,

it seems as melanie2/Roundcube-Plugin-Mobile  no more works with new beta version 1.3.0 of Roundcube.
Could you please modify melanie2/Roundcube-Plugin-Mobile for new Roundcube version?

Thank you a lot!

**Comment by tpayen** [comment]
I look asap, but as it's a beta it will surely wait for the rc

**Comment by gnadelwartz** [comment]
Hi,

the rc is out since april and final can be expected in June.
Any thing new about compatibility with RC 1.3?

**Comment by tomsommer** [comment]
Any ETA?

**Comment by tpayen** [comment]
The main problem is that this mobile skin is based on JQuery Mobile which doesn't seems maintained since a while, and as Roundcube 1.3.0 use a new JQuery version (3.X), this is incompatible with the last version of JQuery Mobile (1.4.5).
I'll have to find a solution for this or cross the fingers for a new JQuery Mobile version until the official Roundcube elastic skin is out

**Comment by gnadelwartz** [comment]
https://blog.jquerymobile.com/2017/05/

since Mai/17 JQ Mobile 1.5 alpha is availible, is this compatible with JQ3.x?

**Comment by tpayen** [comment]
Indeed, this alpha version is available since January, unfortunately it doesn't seem compatible with the JQuery version used in Roundcube 1.3.0. And the fact that the alpha version has not changed since January does not encourage optimism

**Comment by guylando** [comment]
jquery mobile 1.5 is compatible with jquery 3: https://github.com/jquery/jquery-mobile/issues/8546#issuecomment-271163317

**Comment by gnadelwartz** [comment]
See also blog.jQuerymobile.com

**Comment by tpayen** [comment]
Indeed, the version is advertised as compatible. Unfortunately it's not as simple as a change of version otherwise it would already be done. 
I tried the 1.5 alpha version of jquery mobile and it doesn't work better.
I've to take the time to work on this problem, but time is missing. The only solution for now is to use a RC 1.2.5

**Comment by lv01jsi** [comment]
Here is a fix of .js files for mobile skin while RoundCube 1.4 Elastic is coming...
Tested with RC 1.3.4.
Extract to corresponding directories...
[melanie2_larry_mobile_fix_for_RC1.3.zip](https://github.com/messagerie-melanie2/Roundcube-Plugin-Mobile/files/1649224/melanie2_larry_mobile_fix_for_RC1.3.zip)

**Comment by Nippey** [comment]
Hi @lv01jsi,
thank you for the patch!
I tested it with both 1.3.3 abd 1.3.4
I tried it and everything works, but the visuals of the Folder-view seem to be broken (see attached image).

Thanks anyway, this is better than the desktop view ;)

![screenshot_20180122-085633](https://user-images.githubusercontent.com/2164664/35231172-647133a2-ff98-11e7-811f-a27238251b00.png)

**Comment by gnadelwartz** [comment]
For me ist not working in the browser, I got this error after switching to modilw view (note the missing `https:` ...

```
FILE NOT FOUND

The requested resource was not found!
Please contact your server-administrator.
Failed request:
//mail.XXX.de/roundcube/
```

**Comment by gnadelwartz** [comment]
@Nippey had the same issue, solved it be giving read access to all files in `skins/melanie2_larry_mobile`

![image](https://user-images.githubusercontent.com/4593242/35246261-66ae082e-ffc6-11e7-8324-08a3d1bd7853.png)

do not forget to download and unzip https://github.com/messagerie-melanie2/Roundcube-Skin-Melanie2-Larry-Mobile/archive/master.zip in case its not already there

**Comment by tpayen** [comment]
@lv01jsi wow! thank you for that, it works well on rc 1.3.4! Do you want to make a PR to appear as a contributor?

@Nippey it can come from your layout configuration. If you use "widescreen" it breaks the list of messages, I have to update the mobile plugin to force the non-widescreen configuration on mobile.

**Comment by Nippey** [comment]
@tpayen : Awesome, that's it!
Thank you!

I created a second account in roundcube pointing to the same inbox.
This way I separate the settings for "Desktop" and "Mobile" ;)

**Comment by judexzhu** [comment]
@tpayen : Thank you so much !

Could you please tell me how can I set as you mentioned ?

> I have to update the mobile plugin to force the non-widescreen configuration on mobile.

and I'm running the roundcube 1.3.0, it looks has a white stripe on the bottom , anyway I can fix it ?
![image](https://user-images.githubusercontent.com/17834225/35786722-21e3edbe-0a64-11e8-9fa9-931cec76d76a.png)

**Comment by tpayen** [comment]
@judexzhu you can use new releases 0.4.8 for mobile plugin and melanie2_larry_mobile skin, with v1.1 of jquery_mobile plugin. They implement the patch of lv01jsi for RC 1.3 and fix the "layout" problem (and probably the new message button display problem too).

**Comment by fnxweb** [comment]
Kudos, guys! Great job.

**Comment by OpenAai** [comment]
This is great guys! Thanks ever so much!
:-)

**Comment by tharick** [comment]
Great job guys, thanks a lot :D

**Comment by OpenAai** [comment]
RC 1.35 seems to have a problem. Mobile on it's own works, however mobile discovery isn't.
